Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1858,6 +1864,19 @@ + (void)maybePerformActions:(NSArray *)whenConditions
     }
 }
 
++ (NSNumber *)fetchCountDownForContext:(LPActionContext *)actionContext
+{
+    NSDictionary *messageConfig = [LPVarCache sharedCache].messageDiffs[actionContext.messageId];
+    NSNumber *countdown = messageConfig[@"countdown"];
```

Comment:
what is the countdown exactly? is this the delay for the message before it shows?

Reply:
countdown is the time in seconds that server sends based on the trigger.